### DISCUSSION TITLE
Welcome to xtbreak Discussions!

### DISCUSSION THREAD

**Discussion by JanDitzen** [JanDitzen/xtbreak#4] [Announcements]
## 👋 Welcome and thank you for your interest in xtbreak.
Do you have any question, spotted bugs or similar? Feel free to post them here!

**Comment by YikeWangUCD** [comment]
Hi Jan Ditzen, I'm having some problems with xtbreak and would like to ask you. :) According to your documentation, xtbreak model results should be stored in e(breaks),r(CI). However, after building the model, I input e(breaks),r(CI) but it reports an error saying "
breaks not found r(111);". I don't know why this is. Could you please help me to implement the storage of model results? I need this because I intend to run multiple xtbreak models using loops. Thank you so much!

**Comment by JanDitzen** [comment]
Dear YikeWang,
thank you very much for your interest in xtbreak.

It is possible that there is an error in the help file, results might be stores in e(breaks) and e(CI). If you type after running xtbreak "return list" and "ereturn list" you should see the results.

Hope this helps.

Best,

Jan

**Reply by YikeWangUCD** [reply]
Super thanks Jan! It works! Hope you have a lovely day :)
